src/api/task/repository/task_repository.go

```diff
@@ -27,5 +27,21 @@ func (t TaskRepository) Create(_ context.Context, task *model.Tasks) (*model.Tas
 		return nil, err
 	}
 
+	return task, nil
+}
+
+func (r TaskRepository) Update(_ context.Context, task *model.Tasks) (*model.Tasks, error) {
+	err := r.DB.Model(task).Updates(task).Error
+	if err != nil {
+		log.Printf("Error updating Project %v", err)
```

Comment:
Why is the log output "updating Project" ?

Reply:
Ay, el copy paste!